src/runtime/rocm/rocm_device_api.cc

```diff
@@ -200,5 +200,28 @@ TVM_REGISTER_GLOBAL("device_api.rocm").set_body([](TVMArgs args, TVMRetValue* rv
   DeviceAPI* ptr = ROCMDeviceAPI::Global();
   *rv = static_cast<void*>(ptr);
 });
+
+TVM_REGISTER_GLOBAL("profiling.timer.rocm").set_body_typed([](TVMContext ctx) {
+  hipEvent_t start;
```

Comment:
is it safe to copy these?

Reply:
The assumption here is that each function is called only once. If that constraint is violated, then its possible to double free.